.eslintrc

```diff
@@ -17,6 +17,7 @@ plugins:
     - jsx-a11y
     - loosely-restrict-imports
     - react
+    - react-hooks
```

Comment:
Not that this is a bad idea, but are we using React Hooks?

Reply:
I added this more as a future check. Hooks was just released with react v16.8. A new developer can check out our library and make/change a component using hooks.
Obviously we have to approve it :)